app/Http/Controllers/TaskController.php

```diff
@@ -177,5 +177,17 @@ public function archived(Request $request)
    
     }
 
+    public function filter(Request $request){
+        $parameter = $request->filter;
```

Comment:
This is a team environment, using a parameter a field input key as  "filter" is too ambiguous.